app/pods/components/player/player-header/component.js

```diff
@@ -14,6 +16,25 @@ export default class PlayerHeader extends Component {
     }
   }
 
+  @action
+  toggleBookmark() {
+    if(this.content.get('bookmark.id')) {
+
+      this.api.request(`/bookmarks/${this.content.get('bookmark.id')}`, {
```

Comment:
Wrap this in an ember-concurrency task
wait for request's response before updating UI